e2e/tests/LoggedIn/livePublication.e2e.spec.ts

```diff
@@ -104,24 +104,24 @@ test.describe('Live Publication', () => {
 
     test('Author profile', async ({ browser }) => {
         const page = await Helpers.users.getPageAsUser(browser);
-        await page.goto(`/publications/cl3fz14dr0001es6i5ji51rq4`, { waitUntil: 'domcontentloaded' });
+        await page.goto(`/publications/publication-user-6-hypothesis-1-live`, { waitUntil: 'domcontentloaded' });
```

Comment:
A lot of tests involved the octopus user as an example of a normal user - needed to change this to a different one now that octopus is an organisational user and its profile page became slightly different.